src/components/Routes/LayoutRouter/LayoutRouter.tsx

```diff
@@ -0,0 +1,14 @@
+import { Outlet } from 'react-router-dom';
+import { FC } from 'react';
+import { MainNav } from '../../MainNav';
+
+export const LayoutRouter: FC = () => (
+  <>
+    <MainNav />
+    <div className="section">
+      <div className="container">
+        <Outlet />
+      </div>
+    </div>
```

Comment:
You don't have to use this container in layout when you use it on pages

```suggestion
    <MainNav />
    <div className="section">
        <Outlet />
    </div>
```